src/workspaces/ShareWorkspaceModal/Collaborator.test.ts

```diff
@@ -206,4 +206,83 @@ describe('a Collaborator component', () => {
     fireEvent.click(writterPermission);
     expect(setAcl).toHaveBeenCalledTimes(1);
   });
+
+  describe('only allows owners and project owners to share with additional permissions', () => {
+    const setAcl = jest.fn();
+    const aclMock = asMockedFn(setAcl);
```

Comment:
Nit: there's no need for this line. `setAcl` is already typed as a mocked function because it was inferred from `jest.fn()`.

Generally, `asMocked` function is useful for imports that are mocked using `jest.mock`, where TS sees the original type from the import.